microcosm_flask/audit.py

```diff
@@ -28,13 +28,15 @@ def audit(func):
     @wraps(func)
     def wrapper(*args, **kwargs):
         response = None
+        request_id = request.headers.get('X-Request-Id', 'None')
```

Comment:
FYI, we now have code depending on the `audit` function. We can either fix whatever breaks when we change this or do something a little less clean, such as using `setattr` to pass the `should_log_headers` func to `audit`.